Fixed the null on name bug in Google OAuth2

rish07 opened the pull request:
1. The bug was because the user data wasn't being fetched anywhere after logging in from the Firestore but was used in the widget tree, so I have fixed this problem by fetching it on Dashboard.
2. My github auth code was commented in the latest commit in the master branch, is there any bug in it?

Fixes #275

rish07 committed:
Fixed the null on name bug in Google OAuth2

himanshusharma89 commented:
@rish07 Good and no issues in GitHub Auth, I added the ClIent data in the secret file and added it in gitignore, which is causing issue to the new contributors and with the GitHub Action. as the import is invalid now. If you have a better approach to handle the secrets, I am all ears.

rish07 commented:
> @rish07 Good and no issues in GitHub Auth, I added the ClIent data in the secret file and added it in gitignore, which is causing issue to the new contributors and with the GitHub Action. as the import is invalid now. If you have a better approach to handle the secrets, I am all ears.

@himanshusharma89 sure, I'll reply if I can get a solution.
